samples/unity_v230_k8s_121.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: storage.dell.com/v1
```

Comment:
Please remove sample files related to v2.0.0

Reply:
taken care